contrib/aws/gen-json.py

```diff
@@ -101,9 +101,6 @@
 # are started
 data['coreos']['units'] = new_units + data['coreos']['units']
 
-# configure etcd to use its EBS volume
-data['coreos']['etcd']['data-dir'] = '/media/etcd'
```

Comment:
Are we setting this somewhere else? We still need to use the volume mounted at `/media/etcd` for etcd, even if it's bind-mounted into the container.